packages/date-parser/src/index.test.js

```diff
@@ -867,20 +871,22 @@ describe('dateParser', () => {
   it('discards invalid range, keeps the valid part only', () => {
     let res;
 
-    res = parse('yesterday-today', new Date('2018-06-25T05:00:00+08:00'), { timezoneOffset: 60 });
+    const tzPlus1 = 'Africa/Algiers';
+
+    res = parse('yesterday-today', new Date('2018-06-25T05:00:00+08:00'), { timezone: tzPlus1, output: 'date' });
     expect(res.text).toEqual('yesterday');
-    expect(res.start.moment().format('YYYY/MM/DD')).toEqual('2018/06/23');
-    expect(res.end.moment().format('YYYY/MM/DD')).toEqual('2018/06/24');
+    expect(res.start).toEqual('2018-06-23');
+    expect(res.end).toEqual('2018-06-24');
```

Comment:
hmm I also meant to allow the call site to be able to perform date manipulation/format conveniently. Can the call site do so with the chrono output?

Reply:
Then I think it's better to return the dayjs instance (with the correct timezone setup on the instance). It's much easier to work with dayjs instance than Chrono result

Reply:
I plan to return default result as a dayjs instance in the next PRs.

https://github.com/holistics/js/pull/12/commits/72c26c128da91b87153e3abff7e28cb6d16758f8
https://github.com/holistics/js/pull/12/commits/517e3e80cd50bb66448db90bbd8b52b60af24450

Reply:
can we make that change now? So that it is included in this new breaking new version. Otherwise we will need to make another breaking change in the future

Reply:
Yes, we can. Do you have any concerns regarding the up to now changes? Because next, changing the default result would create many changes in our test files.

Reply:
I haven't looked closely to help you on the decision about the best interface. But my general idea is that putting the major changes within this one major breaking version is better than gradually making many breaking changes. So if you think the change is reasonable, I'd suggest including it in this version. I'm fine with reviewing the changes in the tests